ui/dashboardApp/layout/signin/index.js

```diff
@@ -155,6 +172,19 @@ function TiDBSignInForm({ registry }) {
             </Button>
           </Form.Item>
         </AnimationItem>
+        <AnimationItem>
+          <div className={styles.extraLink}>
+            <Dropdown overlay={menu}>
+              <a
+                className="ant-dropdown-link"
```

Comment:
why not simply `<a>`?

Reply:
user can select `sql mode` or `kvauth mode`.